arch/xtensa/src/common/xtensa_swint.c

```diff
@@ -94,16 +102,79 @@ int xtensa_svcall(int irq, void *context, void *arg)
 
       case SYS_save_context:
         {
+          DEBUGASSERT(regs[REG_A3] != 0);
+          memcpy((uint32_t *)regs[REG_A3], regs, (4 * XCPTCONTEXT_REGS));
+        }
+      break;
```

Comment:
```suggestion
      case SYS_save_context:
        {
          DEBUGASSERT(regs[REG_A3] != 0);
          memcpy((uint32_t *)regs[REG_A3], regs, (4 * XCPTCONTEXT_REGS));
        }
        break;
```
`break` statement should be column-aligned to the closing braces.